KMNavigationBarTransition/UIViewController+KMNavigationBarTransition.m

```diff
@@ -59,12 +60,14 @@ - (void)km_viewWillLayoutSubviews {
     UIViewController *toViewController = [tc viewControllerForKey:UITransitionContextToViewControllerKey];
     
     if ([self isEqual:self.navigationController.viewControllers.lastObject] && [toViewController isEqual:self]) {
-        [tc containerView].backgroundColor = [UIColor whiteColor];
```

Comment:
To meet your needs, you can try to modify this line of code to `self.navigationController.navigationBar.superview = [UIColor whiteColor];`